### DISCUSSION TITLE
能不能在Windows中构建MagiskOnWSALocal？

### DISCUSSION THREAD

**Discussion by Sunrongguo2008** [LSPosed/MagiskOnWSALocal#283] [Q&A]
不常用Linux，希望在Windows中构建MagiskOnWSALocal

**Comment by Doddddd** [comment]
https://apps.microsoft.com/store/detail/windows-subsystem-for-linux/9P9TQF7MRM4R

**Comment by Howard20181** [comment]
本来有计划做的，用qemu开个Ubuntu base的虚拟机
但是鸽了
